packages/expo-localization/src/Localization.types.ts

```diff
@@ -65,3 +65,134 @@ export type Localization = {
    */
   timezone: string;
 };
+
+export type Locale = {
+  /**
+   * An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag) with a region code.
+   * @example `'en-US'`, `'es-419'`, `'pl-PL'`.
+   */
+  languageTag: string;
+  /**
+   * An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag) without the region code.
+   * @example `'en'`, `'es'`, `'pl'`.
+   */
+  languageCode: string;
+  /**
+   * The region code for your device that comes from the Region setting under Language & Region on iOS, Region settings on android and is parsed from locale on web (can be null on web).
+   */
+  regionCode: string | null;
+  /**
+   * Currency code for the locale.
+   * @example `'USD'`, `'EUR'`, `'PLN'`.
+   * Returns `null` on web, use a table lookup based on region instead.
+   */
+  currencyCode: string | null;
+  /**
+   * Currency symbol for the locale.
+   * @example `'$'`, `'€'`, `'zł'`.
+   * Returns `null` on web, use a table lookup based on region (if available) instead.
```

Comment:
```suggestion
   * Returns `null` on Web, use a table lookup based on region (if available) instead.
```

Reply:
Can you use `@return` there instead?

* https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#general-approach

Reply:
@Simek It doesn't really seem to show for property fields in `Types` section. Also I think the wording was incorrect in the first place, it's a property not a function return. I reworded it to use `is` instead of `returns`, what do you think?